client/src/document/organisms/sidebar/index.scss

```diff
@@ -99,6 +99,7 @@
     padding: 0.25rem;
     color: var(--text-secondary);
     hyphens: auto;
+    max-width: calc(100% - 1rem);
```

Comment:
The only thing is that I'm not sure what value this should be. To be most exact this should be the width of the marker (which isn't exactly specified) and the actual width rendered probably will differ based on font size, browser, etc.

Reply:
`0.5rem` should be enough. However this won't work for non-link list items (there are some, e.g. undocumented JS properties). @kenrick95 could you look at https://github.com/mdn/yari/pull/5376 that should also fix it. Happy for you to adopt it in this PR and we land yours.